src/__tests__/createActions-test.js

```diff
@@ -0,0 +1,29 @@
+import { createActions } from '../';
+
+describe('createActions()', () => {
+  describe('resulting action creator map', () => {
+    const types = {
+      ACTION: 'ACTION',
+      MY_LONG_ACTION: 'MY_LONG_ACTION'
+    };
+
+    it(`apply a default string transformation function to action types
+        when mapping to action creators`, () => {
+      const actionCreators = createActions(types);
+
+      expect(actionCreators).to.have.property('action');
+      expect(actionCreators).to.have.property('myLongAction');
+    });
```

Comment:
`camelize` is just an arbitrary function that assumes that you're using the conventions of `UPPERCASE_UNDERSCORE_CONSTANT_NAMES`. 
If you're using a different convention you're free to pass your own transformation function.